.github/workflows/test-cron.yaml

```diff
@@ -121,19 +119,23 @@ jobs:
       name: Upload pants.log
       uses: actions/upload-artifact@v3
       with:
-        name: pants-log-bootstrap-linux
+        name: pants-log-bootstrap-Linux-x86_64
         path: .pants.d/pants.log
     - name: Upload native binaries
       uses: actions/upload-artifact@v3
       with:
-        name: native_binaries.${{ matrix.python-version }}.${{ runner.os }}
+        name: native_binaries.${ matrix.python-version }.Linux-x86_64
```

Comment:
We can't use `"${{ runner.os }}-${{ runner.arch }}"` because the M1 runner is an X86 binary running under Rosetta, and it sees its arch as x86_64... But since we know the platform we're targeting, we can just inject that into the string direectly.